metadata-builders/src/main/java/com/linkedin/metadata/builders/search/DatasetIndexBuilder.java

```diff
@@ -125,6 +143,8 @@ private DatasetDocument getDocumentToUpdateFromAspect(@Nonnull DatasetUrn urn,
         return getDocumentToUpdateFromAspect(urn, aspect.getUpstreamLineage());
       } else if (aspect.isGlobalTags()) {
         return getDocumentToUpdateFromAspect(urn, aspect.getGlobalTags());
+      } else if (aspect.isGlossaryTerms()) {
+        return getDocumentToUpdateFromAspect(urn, aspect.getGlossaryTerms());
```

Comment:
we will also need to index glossary terms applied to schema fields- refer to https://github.com/linkedin/datahub/pull/2491 for an example of how to do that

Reply:
Can we cover this in another PR?